README.md

```diff
@@ -5,30 +5,36 @@ Gardener delivers fully managed Kubernetes clusters as a service and provides a
+![](images/gardener-dashboard.gif)
+
+In essence, Gardener is an [extension API server](https://kubernetes.io/docs/tasks/access-kubernetes-api/setup-extension-api-server) that comes along with a bundle of custom controllers. It introduces new API objects in an existing Kubernetes cluster (which is called **garden** cluster) in order to use them for the management of end-user Kubernetes clusters (called **shoot** clusters). These shoot clusters are described via [declarative cluster specifications](https://github.com/gardener/gardener/blob/master/example/90-shoot.yaml) which are observed by the controllers and their control planes are hosted on dedicated **seed** clusters. They will bring up the clusters, reconcile their state, perform automated updates and make sure they are always up and running.
 
 ![](images/gardener-architecture.png)
 
-## Motivation behind the integration of Loki
-As of today [#Gardener](https://gardener.cloud/) manages thousands of Kubernetes clusters SAP and the demand is still growing inside and outside. [#Gardener](https://gardener.cloud) clusters come with a monitoring and logging stack for the control planes of these clusters. While the monitoring stack is straight-forward, the logging stack was more of a challenge as it had to come with these requirements:
+## Motivation
+Today [Gardener](https://gardener.cloud/) already manages thousands of Kubernetes clusters around the globe for all of SAP’s development, quality as well internal and external production requirements. It also serves other [Community members](https://gardener.cloud/adopter/) ([PingCap](https://gardener.cloud/blog/2020-05/00/), [FI-TS](https://metal-stack.io/blog/01-why-metal-stack/), and more) as their multi-cloud orchestration tool of choice. [#Gardener](https://gardener.cloud) clusters come with a monitoring and logging stack for the control planes of these clusters. While the monitoring stack is straight-forward, the logging stack was more of a challenge as it had to come with these requirements:
 1) Resource efficient (cpu, memory)
 2) Secure and reliable
 3) Multi-tenant (per cluster)
 4) Dynamic vertical scaling
 
+### Accessing the logs
+[Gardener](https://gardener.cloud/) provides automatically created and configured logging stack for each cluster. The end user has access the logs from controlplane components like `kube-apiserver`, `kube-controller-manager` and `kube-scheduller` via automatically created Grafana dashboards.
```

Comment:
... provides an automatically created ...

The end user has access to the logs ...